musicbot/cogs/music/__init__.py

```diff
@@ -635,69 +634,104 @@ async def _history(self, ctx):
     @commands.cooldown(1, 5, commands.BucketType.guild)
     @commands.command(name='lyrics')
     async def _lyrics(self, ctx, *query: str):
+        """ Search for lyrics of a song """
+        # Check for API key
+        if not (genius_access_token := self.bot.APIkeys.get('genius')):
+            return await ctx.send(ctx.localizer.format_str('{history.title}'))
```

Comment:
history.title?

Reply:
Should be `errors.missing_api_key`

Reply:
oops. copilot probably got the best of me. fixed